lib/ama_validators.rb

```diff
@@ -9,4 +9,8 @@ module AmaValidators
   require 'ama_validators/phone_number_format_validator'
   require 'ama_validators/province_validator'
 
+  module Rails
```

Comment:
we like to serve up all our JS via the asset pipeline, so if it gets pathed under app/assets/javascripts/ it will be minified/compressed by the app, for example:

https://github.com/amaabca/ama_layout/blob/master/app/assets/javascripts/ama_layout/mobile/mobile_menu.js.coffee